DemoServer/AdditionalLanguages/python/multiMapIndexes/multiMapIndexBasic/metadata.json

```diff
@@ -0,0 +1,147 @@
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#querying-many-sources-at-once-with-multimap-indexes"
+                },
+                {
+                    "Title": "Multi Map indexes from the client",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#multimap-indexes-from-the-client"
+                }
+            ]
+        },
+        {
+            "Title": "The Index Entry class",
+            "Slug": "step-2",
+            "DescriptionLines": [
+                "* The IndexEntry class defines the _index-fields per entry_ in the index.",
+                "",
+                "* These fields contain the indexed data from all the collections defined in the index<br> (configured in the next step).",
+                "",
+                "* In this example, only a single index-field _name_ is defined.<br>",
+                "  Collections defined in the index can be queried and filtered by this field."
+                
+            ],
+            "Assets": [
+                {
+                    "Title": "Querying many sources",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/10-static-indexes-and-other-advanced-options#querying-many-sources-at-once-with-multimap-indexes"
+                },
+                {
+                    "Title": "Multi Map indexes from the client",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/learn/inside-ravendb-book/reader/4.0/12-working-with-indexes#multimap-indexes-from-the-client"
+                }
+            ]
+        },
+        {
+            "Title": "The index constructor",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* Use `add_map()` to define the content of the index-field per collection indexed.<br>",
+                "  Data from the different collections is indexed into the _common index-field_ in the IndexEntry class.",
+                "",
+                "* In this example, document properties from Companies & Suppliers collections are indexed into the common index field _'name'_.<br>",
+                "  The document properties indexed are not customized but taken as is.<br>",
+                "  The document properties are case-sensitive, use the exact casing as in the document.",
+                "",
+                "* The index will contain a mapping from any term generated for the 'name' index-field to the relevant document in either the Companies or the Suppliers collection.",
+                "",
+                "* When querying this index, the index-field 'name' can be used to filter the query results and get matching documents from either collection."
```

Comment:
let's also mention `self._index("name", FieldIndexing.EXACT)` explicitly in this step

Reply:
users may get confused what `index()` method does

Reply:
@poissoncorp 

TBH - I think the following code line should Not be in this place in this demo
`self._index("name", FieldIndexing.EXACT)`

It is Not there originally for all the other languages of this demo, 
as it is not directly needed for the multi-map index explanation.